src/components/policy/policy_external/src/cache_manager.cc

```diff
@@ -775,7 +776,15 @@ bool CacheManager::AppHasHMIType(const std::string& application_id,
 
 void CacheManager::Backup() {
   sync_primitives::AutoLock lock(backuper_locker_);
-  DCHECK(backuper_);
+  if (NULL == backup_thread_) {
```

Comment:
@LitvinenkoIra is it regular situation that there is no backup_thread? May be `DCHECK_OR_RETURN_VOID(backup_thread_)`

Reply:
@LitvinenkoIra any updates?